proto/osmosis/gamm/pool-models/stableswap/stableswap_pool.proto

```diff
@@ -64,10 +64,8 @@ message Pool {
     (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
   ];
   // for calculation amognst assets with different precisions
-  repeated uint64 scaling_factor = 7 [
-    (gogoproto.moretags) = "yaml:\"stableswap_scaling_factor\"",
-    (gogoproto.nullable) = false
```

Comment:
I've testing comparing build warnings with this branch vs main locally, the only build warnings this branch suppressed was `Makefile:242: warning: overriding commands for target test-sim-benchmark
sims.mk:61: warning: ignoring old commands for target test-sim-benchmark` which seems to come from sims.mk.

I was curious why we were removing these lines? (Maybe something in my local env that doesn't trigger warnings related to this?)

Reply:
`make proto-all` does this

Reply:
ah gotcha